src/test/java/org/apache/mesos/scheduler/DefaultSchedulerTest.java

```diff
@@ -429,8 +429,8 @@ public void testOfferAcceptedOnce() throws Exception {
         defaultScheduler.resourceOffers(mockSchedulerDriver, Arrays.asList(offer));
         defaultScheduler.awaitTermination();
 
-        // Verify scheduler accepted resources only once
-        verify(mockSchedulerDriver, times(1)).acceptOffers(any(), any(), any());
+        // Verify scheduler accepted resources twice, once for scheduling and once for recovery.
+        verify(mockSchedulerDriver, times(2)).acceptOffers(any(), any(), any());
```

Comment:
Now that we're accepting twice, you may want to change the name of this test.  `testLaunchAndRecovery`?  Almost anything else is better.